packages/desktop-client/src/components/modals/SelectLinkedAccountsModal.tsx

```diff
@@ -244,7 +299,7 @@ function TableRow({
         {chosenAccount ? (
           <Button
             onPress={() => {
-              onSetLinkedAccount(externalAccount, null);
+              onSetLinkedAccount(externalAccount, undefined);
```

Comment:
With the other change to make the second argument optional
```suggestion
              onSetLinkedAccount(externalAccount);
```